src/index.js

```diff
@@ -3,6 +3,16 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import axios from "axios";
+import 'semantic-ui-css/semantic.min.css'
+
+let apiUrl;
+if (process.env.NODE_ENV === "production") {
+  apiUrl = "https://api-el-gaucho-nyheter.herokuapp.com/";
+} else {
+  apiUrl = "http://localhost:3000/api/v1";
```

Comment:
I would make use of `.env` files to set these values. https://create-react-app.dev/docs/adding-custom-environment-variables/